static-site/components/error-message/index.tsx

```diff
@@ -14,7 +14,7 @@ export default function ErrorMessage({
   title: string;
 
   /** the more specific contents of the error message */
-  contents: string;
+  contents: string | React.ReactElement
```

Comment:
Currently, the `contents` are always displayed within a `<p>`. 
Should there be a type check to against `ReactElement` that are no allowed as content within `<p>`?

Reply:
Rather than do that, I'm going to adjust things so that the type of `contents` is consistently `React.ReactElement`.